agent/server.go

```diff
@@ -203,7 +203,7 @@ func grpcHandlerFunc(rpcServer *server, other http.Handler) http.Handler {
 // DefaultDialRPC is a default RPC client factory function.
 // It creates a new client based on address details from the specific serf member.
 func DefaultDialRPC(caFile, certFile, keyFile string) DialRPC {
```

Comment:
Should this accept context now too?

Reply:
I think the `DefaultDialRPC` is called when initializing the agent, but we only want to start the context when we start making `status` requests to other nodes